src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -68,19 +73,7 @@ public boolean hasNote() {
 
     @Override
     public void periodic() {
-        motor.set(mode.getPower());
-
```

Comment:
This should still be here. We need to set the motor.

Reply:
done